app/controllers/obituaries/recent_controller.rb

```diff
@@ -1,5 +1,7 @@
 class Obituaries::RecentController < ApplicationController
   def index
-    @obituaries = Obituary.all.order(created_at: :desc)
+    @obituaries = Obituary.all.order(:created_at)
+    @obituaries_api = ObituarySearch.new.recent_obituaries
+  
```

Comment:
Hopefully I can figure out this javascript stuff so that we can display this in separate tabs.

Reply:
Oops did I mess this up when I add `:desc` to the activerecord call?

Reply:
Sorry, don't actually remember changing this, I will add it back in and see what happens

Reply:
just added it back in and nothing broke, so I am going with it